components/econet/econet.cpp

```diff
@@ -13,14 +13,18 @@ static const uint8_t SRC_ADR_POS = 5;
 static const uint8_t LEN_POS = 10;
 static const uint8_t COMMAND_POS = 13;
 
+static const uint8_t OBJ_NAME_POS = 6;
+static const uint8_t OBJ_NAME_SIZE = 8;
+static const uint8_t FLOAT_SIZE = 4;
```

Comment:
Bit of a nit, but rather than setting this as a variable I think it's cleaner to call `sizeof(float)`.

Reply:
They are different things though. sizeof(float) depends on the architecture of the device that this code is compiled for. But this size of 4 corresponds to the float size of the econet MCU and how it represents the data in its protocol.

Reply:
Done.